tests/models/gpt_neox/test_modeling_gpt_neox.py

```diff
@@ -396,6 +396,68 @@ def test_model_rope_scaling(self):
             torch.testing.assert_close(ntk_sin_long, original_sin_long)
         self.assertTrue((ntk_scaling_rope.inv_freq <= original_rope.inv_freq).all())
 
+    @require_torch_sdpa
+    @slow
+    def test_eager_matches_sdpa_generate(self):
+        """
+        Based on LlamaModelTest.test_eager_matches_sdpa_generate which also
+        overwrites the common test as the test is flaky on tiny models.
+        """
+        max_new_tokens = 30
+
+        tokenizer = AutoTokenizer.from_pretrained("EleutherAI/pythia-1b")
```

Comment:
Bigger Models needed to match in generation. Same as in Llama, not sure if that's an issue.